.nx/workflows/agents.yml

```diff
@@ -21,3 +21,41 @@ launch-templates:
           rustup toolchain install 1.70.0
           # persist cargo bin into PATH
           echo "PATH=$HOME/.cargo/bin:$PATH" >> $NX_CLOUD_ENV
+
+  tailwind-windows-rust:
+    resource-class: 'windows/medium'
+    image: 'windows-2022'
+    init-steps:
+      - name: Checkout
+        uses: 'nrwl/nx-cloud-workflows/v4/workflow-steps/checkout/main.yaml'
+
+      - name: Restore Node Modules Cache
+        uses: 'nrwl/nx-cloud-workflows/v4/workflow-steps/cache/main.yaml'
+        inputs:
+          key: 'package-lock.json|yarn.lock|pnpm-lock.yaml'
+          paths: 'node_modules'
+          base-branch: 'main' # TODO: should this be next?
+
+      # TODO: Is this needed?
+      # - name: Restore Browser Binary Cache
+      #   uses: 'nrwl/nx-cloud-workflows/v3.6/workflow-steps/cache/main.yaml'
+      #   env:
+      #     KEY: 'package-lock.json|yarn.lock|pnpm-lock.yaml|"browsers"'
+      #     PATHS: |
+      #       '../.cache/ms-playwright'
+      #     BASE_BRANCH: 'main'
+
+      - name: Install Node Modules
+        uses: 'nrwl/nx-cloud-workflows/v4/workflow-steps/install-node-modules/main.yaml'
+
+      # TODO: Is this needed?
+      # - name: Install Browsers (if needed)
+      #   uses: 'nrwl/nx-cloud-workflows/v3.6/workflow-steps/install-browsers/main.yaml'
+
+      - name: Install Rust
+        script: |
+          curl --proto '=https' --tlsv1.3 https://sh.rustup.rs -sSf | sh -s -- -y
+          source "$HOME/.cargo/env"
+          rustup toolchain install 1.70.0
+          # persist cargo bin into PATH
+          echo "PATH=$HOME/.cargo/bin:$PATH" >> $NX_CLOUD_ENV
```

Comment:
Is this the same on windows?

Everything except the rust part was copied from: https://github.com/nrwl/nx-console/blob/df7922e36d02145044ad005f7e6ed1fccdc2d9c9/.nx/workflows/agents.yaml